pallets/creditcoin/src/types.rs

```diff
@@ -255,33 +255,19 @@ fn bytes_to_hex(bytes: &[u8]) -> Vec<u8> {
 	hex
 }
 
-macro_rules! strip_plus {
-    (+ $($rest: tt)*) => {
-        $($rest)*
-    }
-}
-
 macro_rules! concatenate {
+	(@strip_plus + $($rest: tt)*) => {
```

Comment:
I haven't seen this before, does `@` have any special meaning?

Reply:
or is it just a visual cue?

Reply:
There's some info about the pattern [here](https://veykril.github.io/tlborm/decl-macros/patterns/internal-rules.html). Pretty much it's just to make it explicit what internal rule you're referring to. Originally `@` was chosen for a reason, but nowadays it's more just convention.